dataflux_core/download.py

```diff
@@ -310,6 +316,9 @@ def dataflux_download(
     storage_client: object = None,
     dataflux_download_optimization_params: DataFluxDownloadOptimizationParams = None,
     threading_enabled=False,
+    retry_factor: float = 1.2,
```

Comment:
Why pass each parameter independently here instead of passing a google.api_core.retry.retry_unary.Retry object?

Reply:
I was worried that might confuse users trying to use just the client library. If that is preferred I'll modify accordingly.

Reply:
since there's a default I think it probably doesn't make things too difficult for the users. @bernardhan33 any opinions?